pkg/monitor/agent.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package monitor
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog"
+
+	agentquerier "github.com/vmware-tanzu/antrea/pkg/agent/querier"
+	"github.com/vmware-tanzu/antrea/pkg/apis/clusterinformation/v1beta1"
+	clientset "github.com/vmware-tanzu/antrea/pkg/client/clientset/versioned"
+	"github.com/vmware-tanzu/antrea/pkg/querier"
+)
+
+type agentMonitor struct {
+	client  clientset.Interface
+	querier agentquerier.AgentQuerier
+}
+
+// NewAgentMonitor creates a new agent monitor.
+func NewAgentMonitor(
+	client clientset.Interface,
+	querier agentquerier.AgentQuerier,
+) *agentMonitor {
```

Comment:
nit: no need to have 4 lines when they are not too long in one line.

Reply:
Updated.